mbed/mbed.py

```diff
@@ -1278,6 +1279,7 @@ def get_cache(self, url):
     def set_cache(self, url):
         up = urlparse(formaturl(url, 'https'))
         if self.cache and up and up.netloc and os.path.isdir(self.path):
+            cpath = 'none'
```

Comment:
This is not right. `cpath = os.path.join(...` should be moved before the `try:` instead